### ISSUE TITLE
autoLintGradle hangs on dwr library

### ISSUE THREAD

**Issue by TomDePauw** [nebula-plugins/gradle-lint-plugin#108]
I know its an old library but, somewhere the plugin started hanging on this library :

compile 'dwr:dwr:1.1.3'

**Comment by DanielThomas** [comment]
@TomDePauw, can you get us a thread dump that demonstrates where the plugin is hanging?

**Comment by felipecsl** [comment]
I've noticed that applying this plugin into a a really large project (500+ modules) caused the Gradle graph resolution phase time to skyrocket. This was observed by simply running `./gradlew test` in the project root would take ~15 minutes to start the tasks. 
By running `jstack` I could observe that Gradle was spending almost the entire time inside `DefaultTaskExecutionPlan#determineExecutionPlan()` and `finalizerTaskPosition()`. 
After attaching a debugger I noticed that the `autoLintGradle` was being added as a finalizer to every single task apparently, which caused the slowdown.
Disabling the plugin immediately dropped this ~15 minutes step down to ~5 seconds.
